app/src/main/res/layout/activity_main.xml

```diff
@@ -6,14 +6,27 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:id="@+id/TextViewWellcomeText"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+    <fragment
+        android:id="@+id/MainContainer"
```

Comment:
take care of the ids name as well, somewhere the id starts with the uppercase and in other places with lowercase. The convention for this is with lowercase. Please make sure you apply this convention for every id in this project